docs/en/query_language/functions/other_functions.md

```diff
@@ -649,20 +666,27 @@ SELECT FlightDate AS k, count() FROM ontime GROUP BY k ORDER BY k
 └────────────┴─────────┘
 ```
 
-In this selection, the index is not used and ClickHouse processed the entire table (`Processed 4.28 million rows`). To apply the index, select a specific date and run the following query:
+To apply the index, select a specific date.
+
+Query:
 
 ```sql
 SELECT FlightDate AS k, count() FROM ontime WHERE k = '2017-09-15' GROUP BY k ORDER BY k
 ```
+
+By using the index, ClickHouse processed a significantly smaller number of rows (Processed 32.74 thousand rows).
```

Comment:
`Processed...` needs to be wrapped in backticks or other quotes like it was few paragraphs above